src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature

```diff
@@ -0,0 +1,44 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Validate Creation for Absence Event Category Descriptors
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When sending a POST request with valid body to
+                  | Name          | Endpoint                               |
+                  | academicWeeks | /ed-fi/absenceEventCategoryDescriptors |
+             Then it should respond with 201
+              And the record can be retrieved with a GET request
```

Comment:
In Esteban's current work, I discouraged him from putting the exact body here. But for your "CRUD" tests, it would be valuable to confirm that we get the expected response. Your line 15 seems good as it is. Assuming that the programmer can save the body from the "When" step, they can refer to it again in the "Then" step. So I think this is good. Suggestion: add a comment on the next line: `# TODO: when writing the step definition, be sure to validate that the response matches the original payload, except for the addition of the 'id' field`